tests/lib/rules/newline-before-return.js

```diff
@@ -111,78 +111,149 @@ ruleTester.run("newline-before-return", rule, {
 
     invalid: [
         {
+            code: "function a() {\nvar b; return;\n}",
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nvar b; \nreturn;\n}"
```

Comment:
Is the idea to rely on multi pass fixing to add the blank newline for this case? If so, I wonder if we could make the fixer method calculate this so it can do it in one pass?